--- ./deps/antidote_crdt/src/antidote_crdt.erl	2024-07-09 18:11:24
+++ ./patches/antidote_crdt.erl	2024-07-09 18:12:33
@@ -73,6 +73,7 @@
 -type internal_crdt() :: term().
 -type internal_effect() :: term().
 
+-export_type([typ/0]).
 -export([is_type/1, alias/1, new/1, value/2, downstream/3, update/3, require_state_downstream/2, is_operation/2]).
 
 % Callbacks implemented by each concrete CRDT implementation
